hazelcast/src/test/resources/com/hazelcast/client/console/testjob-with-hazelcast-codebase/src/main/java/com/hazelcast/jet/testjob/HazelcastBootstrap.java

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
How about shading the hazelcast dependency rather than adding this dummy class as a user class? I think we will test a more real scenario using a jar shading the hz dependency.

Reply:
In that case, I would consider building the jar during the build/tests, otherwise, we would put large a jar into the git repo.

Reply:
With this in mind, I withdraw this request.

Reply:
>  we would put large a jar into the git repo.

Yes, that was the main reason I've decided to mimic the class.